### PR TITLE
Feature/show gender and website on profile page #183

### PR DESCRIPTION
**Pull request by nmate980829**

### CODE REVIEW

File: app/decorators/user_decorator.rb [hunk 1]
```diff
@@ -73,6 +73,25 @@ def link_with_compact_name(options = {})
     link_to compact_name, profile_path(user.screen_name), options
   end
 
+  def gender
+    return if user.gender.blank?
+    return unless Privacy.for(user, 'GENDER').visible
+    return if user.gender == "NOTSPECIFIED"
```
Comment (OmTheTurtle) [inline app/decorators/user_decorator.rb:79]: I'd combine these three `if`s into one.

File: app/decorators/user_decorator.rb [hunk 2]
```diff
@@ -73,6 +73,25 @@ def link_with_compact_name(options = {})
...
+    genders = {
+      "MALE"  => "Férfi",
```
Comment (SepsiLaszlo) [inline app/decorators/user_decorator.rb:81]: I think the usage of `'` and `" ` is a bit inconsistent in this file. If there are no interpolation we should use `'`.

Comment (falusi94) [reply app/decorators/user_decorator.rb:81]: I agree, please try to use rubocop, its configuration could be improved, but in the most of the cases it's useful.

File: app/decorators/user_decorator.rb [hunk 3]
```diff
@@ -73,6 +73,25 @@ def link_with_compact_name(options = {})
...
+      "FEMALE" => "Nő",
+      "UNKNOWN" => "Egyéb"
+    }
+    info_box(genders[user.gender], 'uk-icon-info')
```
Comment (falusi94) [inline app/decorators/user_decorator.rb:85]: This mapping is already set in the [application.rb](https://github.com/kir-dev/pek-next/blob/master/config/application.rb#L61) under `config.x.genders`.

What do you think about a solution like
```suggestion

    decorated_gender = Rails.configuration.x.genders[user.gender]
    info_box(decorated_gender, 'uk-icon-info')
```

**Comment by nmate980829** [comment]
> If I'm right you did all the changes. I recommend using the 're-request review' button next time

I'm sorry. Next time i will make sure to do so. I also started using rubocop as you recommended.

**Pull request merged**